src/emulator/cpu.ts

```diff
@@ -264,6 +274,14 @@ export class Cpu {
                 this.state.p = (this.state.p + instruction.len) & 0xffff;
                 return instruction.cycles;
 
+            case Operation.halt:
```

Comment:
I would like to remove that one (or to handle it as `NOP`) for now. Implementing this properly is nontrivial, and it is not used in Tetris according to the disassembly.